spec/controllers/api/v1/materials_controller_spec.rb

```diff
@@ -739,5 +844,116 @@ def validate_included_metadata(metadata_json, metadata)
         expect(new_metadatum.value).to eq(metadatum.value)
       }
     end
+
+    it 'should add parents to a material' do
+      @material = create(:material)
+      parents = create_list(:material, 3)
+
+      @material_json = {
+          data: {
+              relationships: {
+                  parents: {
+                      data: parents.map { |parent| {
+                          id: parent.uuid
+                      }}
+                  }
+              }
+          }
+      }
+
+      update_material
+      expect(response).to be_success
+
+      put_response = response
+      get api_v1_material_path(@material.uuid)
+      get_json = response
+      expect(put_response.body).to eq(get_json.body)
+
+      new_material = Material.find(@material.id)
+
+      expect(new_material.parents).to eq(parents)
+    end
+
+    it 'should append new parents to the existing array' do
+      @material = create(:material_with_parents)
+      parents = create_list(:material, 3)
+
+      @material_json = {
+          data: {
+              relationships: {
+                  parents: {
+                      data: parents.map { |parent| {
+                          id: parent.uuid
+                      }}
+                  }
+              }
+          }
+      }
+
+      update_material
+      expect(response).to be_success
+
+      put_response = response
+      get api_v1_material_path(@material.uuid)
+      get_json = response
+      expect(put_response.body).to eq(get_json.body)
+
+      new_material = Material.find(@material.id)
+
+      expect(new_material.parents).to eq(@material.parents + parents)
```

Comment:
I'd add another check for the parents and expect their children is the @material entity